File: scripts/tsprobe/snippets/round2.lua
local M = {}
M.a = function() end
local x = { [1] = function() end }
function M.outer() local function inner() end; return inner end
print(#M, M[1], M["a"], M.a)
goto done
::done::
break_me = [==[ level two ]==]

File: scripts/tsprobe/snippets/round2.java
package p;
public class Outer { class In { private void pm() {} protected void pr() {} void pk() {} public void pu() {} }
  void m() { class Local { void lm() {} } Runnable r = new Runnable() { public void run() {} }; }
  public static void main(String[] a) { outer: while (true) { continue outer; } }
}
